kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -12,7 +12,7 @@
         :title="title"
         :removeNavIcon="isAppContext && isTouchDevice"
         type="clear"
-        textColor="white"
+        textColor="black"
```

Comment:
Until we add full theming support to Keen UiToolbar, we need to use either 'white' or 'black'. This will be done later on as part of KDS maintenance.